### ISSUE TITLE
Migration testing in `next` branch is not handling rollback with new migrations

### ISSUE THREAD

**Issue by gsnider2195** [nautobot/nautobot#3435]
### Environment
* Nautobot version (Docker tag too if applicable): 2.0.0alpha
* Python version: 3.10
* Database platform, version: Postgres 13
* Middleware(s):

New migrations introduced for #3431 broke an unrelated test for `dcim/tests/test_migrations`. The ipam migration test still seems to be working in this branch. We may need to circle back on this and discuss changing the way this test is working.

https://github.com/nautobot/nautobot/actions/runs/4430129283/jobs/7771529844

**Comment by bryanculver** [comment]
@nautobot/core are we still performing these? Is #3960 a better way to execute these tests?

**Comment by lampwins** [comment]
@nautobot/core ^ is this needed?

**Comment by glennmatthews** [comment]
IMHO we should rip these tests out entirely since they're broken and unlikely to be fixed - we should proceed with #3279 instead.

**Comment by gsnider2195** [comment]
I agree with @glennmatthews. We should probably keep this issue around to track removing those tests
